modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -86,7 +84,8 @@ const OpenModalOverlay = createSubcomponent('div')({
   displayName: 'Modal.OpenOverlay',
   modelHook: useModalModel,
   elemPropsHook: useModalOverlay,
-})<ModalOverlayProps>((elemProps: ModalOverlayProps, Element, model) => {
+  /* @ts-ignore */
```

Comment:
The `@ts-ignore` is a red flag. It should be removed or there should be a comment why it is needed. The only change I can see is around `children`. What are the errors?